CIScripts/Test/TestRunner.ps1

```diff
@@ -103,3 +103,36 @@ function Run-Tests {
         }
     })
 }
+
+function Run-Tests {
+    Param ([Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions)
+
+    try {
+        Run-TestsWithoutLogs -Sessions $Sessions
+    }
+    catch {
```

Comment:
Hmm.. now that I think about it, maybe we should print logs even if build passes? That is to say, replace `catch` with `finally`? @krdln @PawelMarchewkaCodiLime thoughts?

Reply:
Not sure about that. I don't see a reason to print, but I also don't see a reason not to print.

If we've stored these logs somewhere, we could store them unconditionally, but print conditionally?

Reply:
nvm, driver logs are old

Reply:
Appending would require modifying docker driver probably. And yeah, there are many invocations of docker driver and showing last of them doesn't really make sense in case when everything passes